lib/cosmos/tools/cmd_tlm_server/api.rb

```diff
@@ -120,7 +120,6 @@ def initialize
         'stop_raw_logging_router',
         'get_server_message_log_filename',
         'start_new_server_message_log']
-      @mutex = Mutex.new
```

Comment:
This mutex causes "no _dump_data is defined for class Thread::Mutex" errors when the CTS starts. I think it's because I declare it last and it's getting passed back (by default) somewhere when we're trying to Marshall everything and calculate the MD5 sum. In any case I've already seen the JRuby test fail even with this "fix" so I'm removing it.